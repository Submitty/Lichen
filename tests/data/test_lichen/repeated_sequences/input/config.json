{
    "semester": "f21",
    "course": "plagiarism",
    "gradeable": "repeated_sequences",
    "config_id": "1",
    "version": "all_versions",
    "regex": "",
    "regex_dirs": [
        "submissions"
    ],
    "language": "plaintext",
    "threshold": 5,
    "sequence_length": 4,
    "prior_term_gradeables": [],
    "ignore_submissions": []
}
