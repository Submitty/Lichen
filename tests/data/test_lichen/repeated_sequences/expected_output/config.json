{
    "semester": "f21",
    "course": "plagiarism",
    "gradeable": "repeated_sequences",
    "config_id": "1",
    "version": "all_versions",
    "regex": [""],
    "regex_dirs": [
        "submissions"
    ],
    "language": "plaintext",
    "threshold": 5,
    "hash_size": 4,
    "other_gradeables": [],
    "ignore_submissions": []
}
